README.md

```diff
@@ -4,3 +4,20 @@
 ![Cocoapods platforms](https://img.shields.io/cocoapods/p/CellKit.svg)
 ![License](https://img.shields.io/cocoapods/l/CellKit.svg)
 ![Continuous integration](https://img.shields.io/bitrise/57c06040e101a852.svg?token=xvYC9NW6ZszIzFdjkapmVg)
+
+Table View and Collection View data source wrapper
+
+## Requirements
+
+- iOS 9.0+
+- Xcode 7.3
+
+## Installation
+ 
```

Comment:
Could you please just add short information about Swift Package manager support?

Similar to this one (but simpler since this library does not have any subspecs):

https://github.com/thefuntasty/FTTestingKit#installation

Reply:
sorry I didn't see this , if I still can help let me know

Reply:
Sure, just fix the issues I pointed out and then we can merge.